inference-engine/tests/functional/plugin/cpu/single_layer_tests/adaptive_pooling.cpp

```diff
@@ -62,14 +71,14 @@ class AdaPoolLayerCPUTest : public testing::WithParamInterface<AdaPoolLayerCPUTe
         std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
 
         CPULayerTestsDefinitions::AdaPoolSpecificParams adaPoolParams;
-        auto netPrecision = InferenceEngine::Precision::UNSPECIFIED;
+        ElementType netPrecision;
         std::tie(adaPoolParams, mode,  netPrecision, targetDevice) = basicParamsSet;
-        inPrc = outPrc = netPrecision;
         std::tie(pooledSpatialShape, inputShape) = adaPoolParams;
 
         ngraph::Shape coordsShape = {pooledSpatialShape.size() };
         auto pooledParam = ngraph::builder::makeConstant<int32_t>(ngraph::element::i32, coordsShape, pooledSpatialShape);
-        auto params = ngraph::builder::makeParams(ngraph::element::f32, {inputShape});
+        init_input_shapes(inputShape);
+        auto params = ngraph::builder::makeDynamicParams(ngraph::element::f32, { inputDynamicShapes[0] });
```

Comment:
what about tests with 'parameter' second input?

Reply:
Please see the latest commit.